src/legacy/ui/public/doc_title/__tests__/doc_title.js

```diff
@@ -33,13 +33,14 @@ describe('docTitle Service', function () {
   beforeEach(function () {
     initialDocTitle = document.title;
     document.title = MAIN_TITLE;
+    setBaseTitle(MAIN_TITLE);
```

Comment:
Previously, the first use of `Private` on `DocTitleProvider` set the `baseTitle`. Being able to set the baseTitle is only needed in testing as far as I can tell.